IndyAV/simulation/indyav_gazebo/urdf/indyav_car.urdf.xacro

```diff
@@ -15,27 +14,30 @@
     </collision>
   </link>
 
-  <xacro:sylphase name="ins"
-                  xyz="0.05 0.0 0.05" rpy="0 0 0" />
+  <xacro:front_axle name="front_left_axle"  xyz="0.4 -1 -0.2"/>
+  <xacro:front_axle name="front_right_axle"  xyz="-0.4 -1 -0.2"/>
+
+  <xacro:wheel name="front_left_wheel" xyz="0 0 0" parent_link="front_left_axle_link"/>
+  <xacro:wheel name="front_right_wheel" xyz="0 0 0" parent_link="front_right_axle_link"/>
 
-  <xacro:back_wheel name="back_left_wheel"  xyz="-1   0.4 -0.2"/>
-  <xacro:back_wheel name="back_right_wheel"  xyz="-1 -0.4 -0.2"/>
+  <xacro:wheel name="back_left_wheel"  xyz="0.4  1 -0.2"/>
+  <xacro:wheel name="back_right_wheel"  xyz="-0.4 1 -0.2"/>
```

Comment:
Due to how the go-kart is constructed, the back two wheels are directly connected via a straight axle. So we should have them be one link with the two separate inertial and collision geometries now that we are simulating the wheels as rotating.

Reply:
Why would the inertial and collision geometries be different? Also, what functionality would this axle have? I attempted to connect the two wheels with a fixed joint to a back axle, but this caused the tires to rotate at the same rate when turning (not allowing the turn to be completed since tires do not move at the same speed during a turn).

Reply:
* so the reason you could not connect the two wheels is that that will introduce a cycle in the robot, which must be a tree of link and joints.
* Having two independent back wheels in simulation will give the car in simulation a much tighter turning radius than the car in real life. (in cars on the road a slip differential is used to address this problem, but in on the go kart, the back tires just slip a little bit on the pavement)